### DISCUSSION TITLE
How enable Mult-lang on site?

### DISCUSSION THREAD

**Discussion by pasimoes** [CaiJimmy/hugo-theme-stack#459] [Q&A]
how can we enable Mult-lang on site using this theme? 
For example: blog1 write in en, pt-br

**Comment by CaiJimmy** [comment]
This theme does not have currently a language switch menu. You can still use Hugo's multilanguage feature and add manually a link to each language on the sidebar.

Ref: https://gohugo.io/content-management/multilingual/
